core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -415,6 +415,10 @@
   TSERV_COMPACTION_SERVICE_ROOT_PLANNER("tserver.compaction.major.service.root.planner",
       DefaultCompactionPlanner.class.getName(), PropertyType.CLASSNAME,
       "Compaction planner for root tablet service"),
+  TSERV_COMPACTION_SERVICE_ROOT_RATE_LIMIT("tserver.compaction.major.service.root.rate.limit", "0B",
```

Comment:
I worry that we have added *far* too many new configuration properties for compactions, and that this might create a terrible user experience around compactions. I wonder if some of these could be consolidated into a pluggable configuration strategy object that is less complicated to configure.

Reply:
Also, what does "root" refer to here for this property? The description should leave that obvious, rather than require users to refer to the descriptions of other properties that have similar naming schemes in order to guess at what it might mean for this property.

Reply:
> I worry that we have added far too many new configuration properties for compactions, and that this might create a terrible user experience around compactions.

Could open a new issue and discuss a course of action.  

> Also, what does "root" refer to here for this property? 

This is a compaction service for the root table.

Reply:
The previous property that shares the same prefix mentions the root table.

Reply:
> > I worry that we have added far too many new configuration properties for compactions, and that this might create a terrible user experience around compactions.
> 
> Could open a new issue and discuss a course of action.

For the larger issue, yes. But I'm struggling to keep up with the pace of all these changes, and am raising the point here, to step back and ask the more narrow question of whether these specific properties are needed. If they are, that's fine... we can move on. But, I don't know if I have the mental capacity to circle back and form a coherent discussion around the larger issue that a separate ticket would serve. The time to do that was probably several weeks ago before the wave of new properties got introduced, and that ship may have sailed.

> 
> > Also, what does "root" refer to here for this property?
> 
> This is a compaction service for the root table.

Okay, cool. Thanks. However, I suppose the point of my feedback wasn't to get a response here, but to get a response in the form of an improved description field in the code.